geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -227,9 +230,22 @@ public ResultModel createRegion(
       regionConfig.setType(regionShortcut.name());
       regionConfig.setRegionAttributes(
           new RegionConverter().createRegionAttributesByType(regionShortcut.name()));
+      RegionAttributesType regionAttributesType = regionConfig.getRegionAttributes();
```

Comment:
these type of validation can be done in the `preExecution` method on the gfsh side , see logic in the `Interceptor` inner class in the same class filee.

Reply:
Done!